test/integration/verify/controls/google_cloudfunctions_cloud_function.rb

```diff
@@ -0,0 +1,44 @@
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_cloudfunctions_cloud_function resource.'
+
+gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+cloudfunction = attribute('cloudfunction', default: {
+  "name": "inspec-gcp-function",
+  "location": "europe-west1",
```

Comment:
Can we use the global location variable here for consistency with a single chosen region?

Reply:
Ah yeah, so cloud functions only has a certain set of locations that it is available in. See: https://cloud.google.com/functions/docs/locations

So I figure it should be configured separately

Reply:
Sure, that makes sense.  Can we keep this as a non-MM yaml variable for convenience?  Would be good to update the README e.g. adding the new variable such as `GCP_CLOUD_FUNCTION_REGION` to the instructions on changing region.

Reply:
Added to the config file and README

Reply:
Fantastic, thanks @slevenick !

Reply:
Thanks @slevenick.